spec/serializers/receive_application_serializer_spec.rb

```diff
@@ -9,27 +9,31 @@
   let(:data) { JSON.parse(subject)['data'] }
   let(:attributes) { data['attributes'] }
 
-  it 'includes id' do
+  it 'includes :id' do
     expect(data['id']).to eq(receive_application.receive_application_id)
   end
 
-  it 'includes tracking_number as attribute' do
+  it 'includes :type' do
+    expect(data['type']).to eq('preneeds_receive_applications')
+  end
+
+  it 'includes :tracking_number as attribute' do
     expect(attributes['tracking_number']).to eq(receive_application.tracking_number)
   end
 
-  it 'includes submitted_at as attribute' do
+  it 'includes Lsubmitted_at as attribute' do
```

Comment:
lil typo here. should it be this?

```suggestion
  it 'includes :submitted_at as attribute' do
```

Reply:
Nice catch!